tests/data/sample_data/non_hrv_zeros.zarr/.zattrs

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Could you make this data on the fly, rather than committing it? 
In the past we commit these test zarr files but they quickly got out of data, and became annoying to update

Reply:
How about a halfway solution where the coordinates are saved out as a zarr but the data itself is created on the fly?

This means we don't have to have a big list of x and y coordinates, variable names, timestamps, steps in `conftest.py` as well as some attributes we need to append to describe the coordinates. I've just committed this change, maybe have a look? 

I think it makes the conftest cleaner than it would be otherwise, and the saved data is pretty light with a total of 232KB for the NWP and satellite combined